test/models/lookup_value_test.rb

```diff
@@ -144,124 +155,112 @@ def valid_attrs2
+    skip('puppet value test')
+    value = LookupValue.new(:value => nil, :match => "hostgroup=Common", :lookup_key_id => boolean_lookup_key.id, :omit => true)
     assert_valid value
   end
 
   test "lookup value should allow valid key" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "hostgroup=Common", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "hostgroup=Common", :lookup_key_id => boolean_lookup_key.id)
     assert_valid value
   end
 
   test "lookup value should allow valid multiple key" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "hostgroup=Common,domain=example.com", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "hostgroup=Common,domain=example.com", :lookup_key_id => boolean_lookup_key.id)
     assert_valid value
   end
 
   test "lookup value should not allow for nil key" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "", :lookup_key_id => boolean_lookup_key.id)
     refute_valid value
   end
 
   test "lookup value will be rejected for invalid key" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "hostgroup=", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "hostgroup=", :lookup_key_id => boolean_lookup_key.id)
     refute_valid value
     assert_equal "is invalid", value.errors[:match].first
   end
 
   test "lookup value will be rejected for invalid multiple key" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "hostgroup=Common,domain=", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "hostgroup=Common,domain=", :lookup_key_id => boolean_lookup_key.id)
     refute_valid value
     assert_equal "is invalid", value.errors[:match].first
   end
 
   test "lookup value will be rejected for invalid matcher" do
-    key = lookup_keys(:three)
-    value = LookupValue.new(:value => true, :match => "something=Common", :lookup_key_id => key.id)
+    value = LookupValue.new(:value => true, :match => "something=Common", :lookup_key_id => boolean_lookup_key.id)
     refute_valid value
     assert_equal "something does not exist in order field", value.errors[:match].first
   end
 
   test "shouldn't save with empty boolean matcher for smart class parameter" do
+    skip('puppet value test')
     lookup_key = FactoryBot.create(:puppetclass_lookup_key, :key_type => 'boolean', :override => true,
                                     :default_value => "true", :description => 'description')
     lookup_value = FactoryBot.build_stubbed(:lookup_value, :lookup_key => lookup_key, :match => "os=fake", :value => '')
     refute lookup_value.valid?
   end
```

Comment:
Should this (and the other puppet-specific tests) be extracted to a separate file instead of skipped?

Reply:
Yeah, definitelly, let me have another look at those :)

Reply:
I've extracted puppet related tests to separate file to be extracted more easily.